### PR TITLE
fix IE and Android compatibility issues

### PR DESCRIPTION
**Pull request by yahiousun**
Add dependencie: 'babel-runtime'. Check #39 for details.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ module.exports = {
             test: /\.js?$/,
             loader: 'babel',
             query: { 
-              presets: ['react', 'es2015'], 
-              plugins: ['transform-es2015-modules-commonjs', 'transform-object-rest-spread'] 
+              presets: ['react', 'stage-0', 'es2015'], 
+              plugins: ['transform-runtime']
```
Comment (conorhastings) [inline webpack.config.js]: why is this needed?

File: webpack.config.js [hunk 2]
```diff
@@ -17,8 +17,8 @@ module.exports = {
             test: /\.js?$/,
             loader: 'babel',
             query: { 
-              presets: ['react', 'es2015'], 
-              plugins: ['transform-es2015-modules-commonjs', 'transform-object-rest-spread']
```
Comment (conorhastings) [inline webpack.config.js]: why remove commonjs module tplugin?

File: .babelrc [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-	"presets": ["es2015", "react"],
-	"plugins": ["transform-object-rest-spread", "transform-es2015-modules-commonjs"]
+	"presets": ["es2015", "stage-0", "react"],
```
Comment (conorhastings) [inline .babelrc]: same questions as below

File: webpack.config.js (hunk 1 above)
Comment (yahiousun) [inline webpack.config.js]: IE and some android webview doesn't support Object.assign
transform-runtime will fix it

File: .babelrc (hunk 3 above)
Comment (yahiousun) [inline .babelrc]: I use 'babel-preset-stage-0' replace 'babel-plugin-transform-object-rest-spread',
this is not necessary, just in case forgot to add transform plugins.

File: webpack.config.js (hunk 2 above)
Comment (yahiousun) [inline webpack.config.js]: you don't need to spec it, it's in 'babel-preset-es2015'.

File: .babelrc (hunk 3 above)
Comment (conorhastings) [inline .babelrc]: prefer not to use stage 0

**Comment by conorhastings** [comment]
@yahiousun can you also rebase, this will fix the tests

File: package.json [hunk 4]
```diff
@@ -4,17 +4,18 @@
   "description": "sane syntax highlighting component for react",
   "main": "dist/index.js",
   "dependencies": {
+    "babel-runtime": "^6.18.0",
```
Comment (conorhastings) [inline package.json]: I think this should be in devDependencies

File: .babelrc (hunk 3 above)
Comment (yahiousun) [inline .babelrc]: I see, what about add 'babel-plugin-transform-object-assign' or 'babel-plugin-transform-runtime', that will work too :)

Comment (conorhastings) [inline .babelrc]: yeah that would be good

File: package.json (hunk 4 above)
Comment (yahiousun) [inline package.json]: get semaller 

before
```javascript
var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

```
after
```javascript
var _extends2 = require('babel-runtime/helpers/extends');

var _extends3 = _interopRequireDefault(_extends2);
```

Comment (conorhastings) [inline package.json]: cool, just rebase and i can merge
